compiler/qsc/src/compile.rs

```diff
@@ -37,6 +37,9 @@ pub enum ErrorKind {
     #[error("Cycle in dependency graph")]
     /// `DependencyCycle` occurs when there is a cycle in the dependency graph.
     DependencyCycle,
+
+    #[diagnostic(transparent)]
+    OpenQasm(#[from] crate::qasm::error::Error),
```

Comment:
Is this used? All tests pass if I comment it out.

Reply:
It is used in an incoming branch.